cms/updates/0.0.5-evidence-types-update.js

```diff
@@ -0,0 +1,120 @@
+const keystone = require("keystone"),
+	async = require("async"),
+	log4js = require("log4js"),
+	fs = require("fs"),
+	path = require("path");
+
+const logger = log4js.getLogger();
+const evidenceTypeList = keystone.list("EvidenceType");
+
+const deleteEvidenceType = async (evidenceType, done) => {
+	let eTitle = evidenceType.title;
+	logger.info(`Deleting evidence type: ${eTitle}`);
+
+	try {
+		await evidenceTypeList.model.findOne({ title: { "$regex": eTitle, "$options": "i" } }).remove();
+		done();
+	} catch (err) {
+		logger.error(`Error deleting '${eTitle}'`, err);
+		done(err);
+	}
+
+	logger.info(`Successfully deleted '${eTitle}'`);
+};
+
+const updateEvidenceType = async (jsonldEvidenceType, done) => {
+	let masEvidenceType = await evidenceTypeList.model.findOne({key: jsonldEvidenceType["@id"] }).exec();
+	if(!masEvidenceType)
+		return;
+
+	let newTitleShort = jsonldEvidenceType["prefLabel"]["@value"];
+	let newTitle = masEvidenceType.broaderTitle + " - " + newTitleShort;
+	let oldTitle = masEvidenceType.title;
+
+	if(newTitle != oldTitle && !oldTitle.includes(newTitleShort)){
```

Comment:
includes is case sensitive. is that a problem?

how do you know the old title includes the prefLabel value?

Reply:
Dev testing has shown that case sensitivity this is not an issue.The old title only contains the pref lable value if the item's title doesnt need updating. For example, the item with the old title "Guidance and advice - Drug prescribing" and the new title "Drug prescribing" wouldn't be updated as it would strip the "Guidance and advice - " part which isn't what we want.